sdk/core/azure-core/azure/core/tracing/_abstract_span.py

```diff
@@ -202,20 +204,30 @@ class HttpSpanMixin(_MIXIN_BASE):
     _HTTP_METHOD = "http.method"
     _HTTP_URL = "http.url"
     _HTTP_STATUS_CODE = "http.status_code"
+    _NET_PEER_NAME = "net.peer.name"
+    _NET_PEER_PORT = "net.peer.port"
 
     def set_http_attributes(self, request: "HttpRequest", response: Optional["HttpResponseType"] = None) -> None:
         """
         Add correct attributes for a http client span.
 
         :param request: The request made
         :type request: HttpRequest
-        :param response: The response received by the server. Is None if no response received.
+        :param response: The response received from the server. Is None if no response received.
         :type response: ~azure.core.pipeline.transport.HttpResponse or ~azure.core.pipeline.transport.AsyncHttpResponse
         """
         self.kind = SpanKind.CLIENT
         self.add_attribute(self._SPAN_COMPONENT, "http")
         self.add_attribute(self._HTTP_METHOD, request.method)
         self.add_attribute(self._HTTP_URL, request.url)
+
+        host = urlparse(request.url).netloc
```

Comment:
nit: would `.hostname` work instead of splitting on `:` ? Then youll have access to the port via `.port`

Reply:
Ah, yea, that'd be cleaner. Thanks for the suggestion! I didn't realize the urlparse result had `hostname` and `port` attributes since printing the result didn't list those fields. 😅